contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -578,31 +592,31 @@ contract BiconomyTokenPaymaster is
         // This model assumes irrespective of priceSource exchangeRate is always sent from outside
         // for below checks you would either need maxCost or some exchangeRate
 
-        // review: can add some checks here on calculated value, fee cap, exchange rate
         uint256 tokenRequiredPreFund = ((requiredPreFund + costOfPost) *
             exchangeRate) / 10 ** 18;
         require(
             tokenRequiredPreFund != 0,
             "BTPM: calculated token charge invalid"
         );
-        require(
-            fee <= (tokenRequiredPreFund * 20) / 100,
-            "BTPM: fee markup too high"
-        );
+
+        // check for caps after applying markup against without markup
```

Comment:
This comment is not that clear. Can we use more simple language?

Reply:
I am removing the comment. this is meant in the sense you calculate tokenRequiredPreFund first then check flat fee how much percent higher it is and apply limit.

removing comment as this version doesn't have flat fee.